plugin/pki/path_venafi_cert_enroll.go

```diff
@@ -96,6 +110,13 @@ func (b *backend) pathVenafiIssue(ctx context.Context, req *logical.Request, dat
 		return logical.ErrorResponse("role key type \"any\" not allowed for issuing certificates, only signing"), nil
 	}
 
+	ttl := time.Duration(data.Get("ttl").(int)) * time.Second
+	if role.MaxTTL > 0 && ttl > role.MaxTTL {
+
+		return logical.ErrorResponse(errorTextValueMustBeLess), nil
```

Comment:
This behavior (error) is inconsistent with the native Vault PKI.  If a `max_ttl` value is specified for the role and a `ttl` value greater than that is specified when requesting the certificate, then the validity of the Venafi enrolled certificate should be the role `max_ttl`.

Reply:
fixed on the latest commit.